jetcd-osgi/src/main/resources/features.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!--
+
+    Copyright 2017 The jetcd authors
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<features xmlns="http://karaf.apache.org/xmlns/features/v1.4.0" name="coreos-jetcd">
+    <feature name="coreos-jetcd" description="coreos-jetcd" version="${project.version}}">
+
+        <details>CoreOS :: Java Client for etcd V3</details>
+
+        <feature prerequisite="true">wrap</feature>
+
+        <!-- google -->
```

Comment:
why is this commented out? prefer codebase to be as minimum as possible.

Reply:
This is left as documentation to show how the karaf feature should looks like once grpc solves the split package issue

Reply:
i see. Could you add comment to explain the intention?